mods/sample-nginx.config

```diff
@@ -82,27 +82,31 @@ server {
 
   # rewrite to front controller as default rule
   location / {
-    if ($is_args != "") {
-        rewrite ^/(.*) /index.php?pagename=$uri&$args last;
-    }
-    rewrite ^/(.*) /index.php?pagename=$uri last;
+     if (!-e $request_filename) {
+     rewrite ^(.*)$ /index.php?q=$1;
```

Comment:
Please use an indenting consistent with the rest of the file.

Reply:
@annando is the `?q` correct or should it be `?pagename`? I remember some collisions with that parameter which lead to the name change, but I don't remember the direction and the coffee is slow this morning to boot that thing between my shoulders.

Reply:
It should always be ```pagename```. We only are using ```q``` for compatibility reasons. The ```q``` is a parameter that we need in the API for the search. So this would conflict.

Reply:
So @anmol26s should also change the `?q=` into `?pagename=` not only fix the indentation.